ports/servoshell/webview.rs

```diff
@@ -113,6 +124,81 @@ where
         }
     }
 
+    pub fn handle_gamepad_events(&mut self) {
```

Comment:
I think inside the method we can add various links to concepts in https://w3c.github.io/gamepad/#remapping